src/main/java/jenkins/scm/api/SCMSourceOwners.java

```diff
@@ -69,7 +70,7 @@
         public Iterator<SCMSourceOwner> iterator() {
             Jenkins j = Jenkins.getInstance(); // TODO 1.590+ getActiveInstance
```

Comment:
Regardless of the final behavior you select, leaving in a TODO comment referring to a Jenkins version you are now past is a clear mistake. If you intend to retain the null check, then the comment should be deleted.